src/main/java/RacingGame/domain/Car.java

```diff
@@ -32,11 +30,16 @@ public String carName(){
         return this.name;
     }
 
-    public void applyRule(int location) {
-        this.location = location;
+    public void move(Rule rule) {
+        if (rule.move())
```

Comment:
`if` 문의 경우 한줄이라면 `{` 를 생략 가능하겠지만 
이번 과정에서는 연습삼아 좀 더 코딩컨벤션을 엄격하게 준수하시는 걸 추천 드립니다 😄 

```suggestion
        if (rule.move()) {
            location++;
        }
```